src/react/components/connect.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from "react";
 import shallowEqual from "../../utils/shallowEqual";
 import propValidation from "../../utils/propsValidation";
 import bindActions from "../../utils/bindActions";
-import { mapToProps } from "../../interfaces/Helpers";
+type mapToProps = (state: object, ownProps?: object) => object;
```

Comment:
Why did you change this?

Reply:
This is because you don't want to export the mapToProps typing, I had to duplicate in two files.